multipy/runtime/loader.cpp

```diff
@@ -67,6 +67,14 @@
 #include <multipy/runtime/loader.h>
 #include <multipy/runtime/mem_file.h>
 
+/* Linux traditionally doesn't have the trailing 64 that BSD has on these. */
```

Comment:
Is this for CentoOS 7? Can you a bit more clear on why its needed.

Reply:
Great catch, I copied the whole block and forgot to adapt the comment.